src/molecules/formfields/CreditCardField/Readme.md

```diff
@@ -24,7 +24,7 @@ CreditCardField Example:
       securityCode={(
         <TextField
           placeholder='3/4 Digit Security Code'
-          inputTooltip={() => alert('input tooltip clicked')}
+          inputTooltip={'input tooltip clicked'}
```

Comment:
These changes remove the annoying popups that appear on page load

Reply:
you are a hero!

Reply:
![](https://media.giphy.com/media/5xaOcLDE64VMF4LqqrK/giphy.gif)